### ISSUE TITLE
Submitting a book causes worker node process to timeout

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#2853]
@annetteholtkamp reports that just submitting a ~~(Russian)~~ book (with an URL for the abstract) is impossible because the server simply timeouts.

@david-caro and I briefly checked and we found on /var/log/supervisor/inspire-stderr.log: 
```
[2017-10-10 17:20:06 +0000] [1083] [CRITICAL] WORKER TIMEOUT (pid:12197)
[2017-10-10 17:20:07 +0000] [30001] [INFO] Booting worker with pid: 30001
```

We were not able to retrieve further errors messages...

**Comment by kaplun** [comment]
@jmartinm this is the issue. Is there anyone who can look into that?

**Comment by jacquerie** [comment]
Do we have enough to reproduce here? A specific russian book? Any russian book? With what other metadata?

**Comment by kaplun** [comment]
Any metadata. Probably russian is unrelated.

**Comment by jacquerie** [comment]
Can we at least know which book she originally tried?

**Comment by annetteholtkamp** [comment]
These are some books I tried in vain - and them submitted using doctype article

- Annette

2.
Principles of optics<https://inspirehep.net/record/1631710>
Max Born<https://inspirehep.net/author/profile/Born%2C%20Max?recid=1631710&ln=en> (Gottingen U.<https://inspirehep.net/search?cc=Institutions&p=institution:%22Gottingen%20U.%22&ln=en> & Edinburgh U.<https://inspirehep.net/search?cc=Institutions&p=institution:%22Edinburgh%20U.%22&ln=en>), Emil Wolf<https://inspirehep.net/author/profile/Wolf%2C%20Emil?recid=1631710&ln=en> (Rochester U.<https://inspirehep.net/search?cc=Institutions&p=institution:%22Rochester%20U.%22&ln=en>). 1999. 952 pp.
DOI: 10.1017/CBO9781139644181<http://dx.doi.org/10.1017/CBO9781139644181>

  *   References<https://inspirehep.net/record/1631710/references> | BibTeX<https://inspirehep.net/record/1631710/export/hx> | LaTeX(US)<https://inspirehep.net/record/1631710/export/hlxu> | LaTeX(EU)<https://inspirehep.net/record/1631710/export/hlxe> | Harvmac<https://inspirehep.net/record/1631710/export/hlxh> | EndNote<https://inspirehep.net/record/1631710/export/xe>
  *   Link to Fulltext<https://www.scribd.com/doc/23494793/Born-Wolf-1999-Principles-of-Optics-7th-Ed>
  *

Detailed record<https://inspirehep.net/record/1631710?ln=en> - Cited by 6 records<https://inspirehep.net/search?ln=en&p=refersto%3Arecid%3A1631710&sf=earliestdate> - Attribute this paper<https://inspirehep.net/author/claim/action?selection=1631710&confirm=True> - Edit record<https://inspirehep.net/record/edit/?ln=en#state=edit&recid=1631710> - Manage files<https://inspirehep.net/record/managedocfiles?ln=en&recid=1631710>

3.
Optical Coherence and Quantum Optics<https://inspirehep.net/record/1631709>
Leonard Mandel<https://inspirehep.net/author/profile/Mandel%2C%20Leonard?recid=1631709&ln=en>, Emil Wolf<https://inspirehep.net/author/profile/Wolf%2C%20Emil?recid=1631709&ln=en>. 1995. 1194 pp.

  *   References<https://inspirehep.net/record/1631709/references> | BibTeX<https://inspirehep.net/record/1631709/export/hx> | LaTeX(US)<https://inspirehep.net/record/1631709/export/hlxu> | LaTeX(EU)<https://inspirehep.net/record/1631709/export/hlxe> | Harvmac<https://inspirehep.net/record/1631709/export/hlxh> | EndNote<https://inspirehep.net/record/1631709/export/xe>
  *   Cambridge University Press Server<http://www.cambridge.org/zw/academic/subjects/physics/optics-optoelectronics-and-photonics/optical-coherence-and-quantum-optics>; Link to Fulltext<http://www.imsc.res.in/~chandru/home/chandru/Optical_Coherence_and_Quantum_Optics_by_Leonard_Mandel.pdf>
  *

Detailed record<https://inspirehep.net/record/1631709?ln=en> - Attribute this paper<https://inspirehep.net/author/claim/action?selection=1631709&confirm=True> - Edit record<https://inspirehep.net/record/edit/?ln=en#state=edit&recid=1631709> - Manage files<https://inspirehep.net/record/managedocfiles?ln=en&recid=1631709>

4.
Introduction to Modern Optics<https://inspirehep.net/record/1630737>
Grant R. Fowles<https://inspirehep.net/author/profile/Fowles%2C%20Grant%20R.?recid=1630737&ln=en> (Utah U.<https://inspirehep.net/search?cc=Institutions&p=institution:%22Utah%20U.%22&ln=en>). 1968. 333 pp.

  *   References<https://inspirehep.net/record/1630737/references> | BibTeX<https://inspirehep.net/record/1630737/export/hx> | LaTeX(US)<https://inspirehep.net/record/1630737/export/hlxu> | LaTeX(EU)<https://inspirehep.net/record/1630737/export/hlxe> | Harvmac<https://inspirehep.net/record/1630737/export/hlxh> | EndNote<https://inspirehep.net/record/1630737/export/xe>
  *   Link to Fulltext<http://www.fulviofrisone.com/attachments/article/404/Introduction%20to%20Modern%20Optics.pdf>
  *

Detailed record<https://inspirehep.net/record/1630737?ln=en> - Attribute this paper<https://inspirehep.net/author/claim/action?selection=1630737&confirm=True> - Edit record<https://inspirehep.net/record/edit/?ln=en#state=edit&recid=1630737> - Manage files<https://inspirehep.net/record/managedocfiles?ln=en&recid=1630737>

5.
Введение в квантовую механику [Vvedenie v kvantovuyu mekhaniku]<https://inspirehep.net/record/1629506> (In Russian)
Introduction to quantum mechanics
D.I. Blokhintsev<https://inspirehep.net/author/profile/Blokhintsev%2C%20D.I.?recid=1629506&ln=en>. 1944. 486 pp.
Published in ОГИЗ [OGIZ], Moscow, Leningrad, 1944

  *   References<https://inspirehep.net/record/1629506/references> | BibTeX<https://inspirehep.net/record/1629506/export/hx> | LaTeX(US)<https://inspirehep.net/record/1629506/export/hlxu> | LaTeX(EU)<https://inspirehep.net/record/1629506/export/hlxe> | Harvmac<https://inspirehep.net/record/1629506/export/hlxh> | EndNote<https://inspirehep.net/record/1629506/export/xe>
  *   Link to fulltext from Russ. Acad. Sci.<http://nasledie.enip.ras.ru/ras/view/publication/general.html?id=46875411>



On 16 Oct 2017, at 15:43, Jacopo Notarstefano <notifications@github.com<mailto:notifications@github.com>> wrote:


Can we at least know which book she originally tried?

**Comment by michamos** [comment]
Fixed by #2930.
